Coda 323

NaraHita opened the pull request:
Improving the Initial Draft for CODA Vision Page.

NaraHita committed:
Revise our vision and mission for CODA

Updated the document to reflect the new vision and mission of the Open Documentation Academy, including core values and programs.

NaraHita committed:
New contributors pathway

Add graphics to better explain a new contributors pathway.

NaraHita committed:
Minor correction for Issue 323.

Signed-off-by: Tara Nair <narahita211@gmail.com>

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,53 @@
-We create opportunities for people who are new to open source by connecting them with documentation tasks on projects of distinction. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment. With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.
+At CODA, Canonical Open Documentation Academy, we envision a future where quality documentation is the foundation and lifeblood of open-source ecosystems—a world where clear, accessible, and thoughtful content demystifies technology, inviting both newcomers and experienced contributors from every corner of the globe to collaborate, innovate, and ensure open-source thrives for generations to come.
 
+---
```
Can we stick to @degville's version for the heading and introduction?

> # Our ambition for the Open Documentation Academy
> 
> We create opportunities for people who are new to open source by connecting them with documentation tasks on established projects. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment.
> 
> With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,53 @@
+---
 
-## What drives us
+## Our Mission
```
We use sentence case for headings. Please see the [Headings](https://docs.ubuntu.com/styleguide/en/#headings) section of the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en/)

activus-d commented on website/our-vision.md:
Let’s create a new page titled “Our values” for this section, as suggested by @degville in his branch. We can place the new page after the "Our vision" page.

activus-d commented on website/our-vision.md:
As mentioned earlier, we use sentence case for headings. Please see the [Headings](https://docs.ubuntu.com/styleguide/en/#headings) section of the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en/).

Also, it would be nice to have a blank line after every heading. This makes it clearer to read the markdown version.

activus-d reviewed (commented):
Well done, @NaraHita. I’ve added a few comments to better align the document with our writing style guide and to bring the introduction more in line with @degville’s suggestion, as discussed during the Community hour.

NaraHita committed:
Revise vision statements and move values to new file.

Updated the vision and mission statements for clarity and coherence, moving values to new file.

NaraHita committed:
Add our core values and vision for CODA members

NaraHita committed:
Remove vision section from our-values.md

Removed the section detailing the vision for member journeys at CODA.

NaraHita committed:
Expand vision section with member journey milestones

NaraHita committed:
Update vision statement for emphasis

Revised wording to emphasize community building in open source.

NaraHita committed:
Adding our-value to toctree

NaraHita committed:
Change heading level for core values section

NaraHita commented:
Hi @activus-d, I have made the changes you requested. 

I can squash all changes, if necessary, after all edits are done.

Thanks.

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,42 @@
+## Our vision
 
-## What drives us
+__Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__
 
```
We can remove this part.
```suggestion
```

NaraHita replied:
Hi @activus-d, I think a highlighted main point would help readers grasp the essence of the para to come. Can we leave this in for 1 more reviewer to look at?

activus-d replied:
Yes, we can! Another community member will review the PR soon. Great job so far @NaraHita

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,42 @@
-Canonical's Open Documentation Academy, or CODA, aspires to be the centre of open source collaboration through documentation.
 
-We create opportunities for people who are new to open source by connecting them with documentation tasks on projects of distinction. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment. With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.
+---
```
Let's remove all line separators to make it look more like a single document.

NaraHita replied:
Done.

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,42 @@
+__Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__
 
-The world of open source collaboration is diverse. There's no singular vision or definition of the idea, meaning that different people and projects hold different values and practice it differently. A person's introduction to it might not always be compassionate or helpful, and many people have rocky starts. There are a few different explanations for this.
+At CODA—the Canonical Open Documentation Academy—we envision a future where *quality documentation forms the foundation and lifeblood of open-source ecosystems*. We see a world where *clear, accessible, and thoughtful content demystifies technology*. This environment welcomes both newcomers and experienced contributors from every corner of the globe. Together, we commit to build a collaborative a ...
```
Let’s introduce the full name first and put the abbreviation in parentheses. After that, we can use the abbreviation `CODA` throughout the rest of the document.

Also, we can make it more concise by removing the last sentence.

```suggestion
At the Canonical Open Documentation Academy (CODA), we envision a future where quality documentation is the foundation of open-source ecosystems. We see a world where clear, accessible, and thoughtful content demystifies technology and creates an environment that welcomes contributors of all experience levels.
```

NaraHita replied:
The last sentence feels important to me. Can we leave it for one more reviewers to look at. Thnx.

activus-d replied:
That’s fine, @NaraHita. Another community member will review the PR soon. Great job so far!

activus-d commented on website/our-vision.md:
```diff
@@ -1,82 +1,42 @@
-The majority of open source projects out in the world were created by happenstance on collaboration platforms like GitHub and GitLab, with no serious vision for contribution. These projects are often abandoned, so you can't always get meaningfully involved.
 
-Some projects proclaim their open sourcing enthusiastically, but their passion gets the better of their experience. They lack a robust, practical collaboration model. It's often difficult to contribute to these projects because they practice it impulsively or without care.
+---
```
Let's remove all line separators.

NaraHita replied:
Done.

activus-d commented on website/our-vision.md:
We can do without the word "premier" here:

> To build a global community advancing open-source documentation excellence through mentorship and collaboration.
> 
> As a pioneering program focused on the craft of documentation, the Open Documentation Academy:

NaraHita replied:
Done.

activus-d commented on website/our-vision.md:
Let's remove all line separators.

NaraHita replied:
Done.

activus-d commented on website/index.md:
```diff
@@ -40,6 +40,7 @@ The Open Documentation Academy is sponsored by [Canonical Ltd](https://canonical
 :maxdepth: 3
 
 our-vision
```
By default, this uses Heading 1 as the page title in the main side navigation. Since the Heading 1 is long, we can customize it to use a shorter version:

<img width="1354" height="596" alt="Screenshot 2025-11-28 at 12 23 47 PM" src="https://github.com/user-attachments/assets/ec26c07e-4ff4-4b3c-bbb0-6ced2615e63d" />

```suggestion
Our ambition for CODA <our-vision>
```

activus-d reviewed (commented):
Great work on this, @NaraHita. I have made some new suggestions, nothing major, just a few minor points to improve the document. Once you have implemented them, you may need to wait for other community members to review it before it's merged.

NaraHita committed:
Update website/index.md

Co-authored-by: Damilola Oladele <98895460+activus-d@users.noreply.github.com>

NaraHita committed:
Revise vision and mission statements in our-vision.md

Updated the vision and mission statements for clarity and consistency.

shaloo commented on website/our-vision.md:
```diff
@@ -1,82 +1,36 @@
-We create opportunities for people who are new to open source by connecting them with documentation tasks on projects of distinction. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment. With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.
+## Our vision
 
+__Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__
```
Accessibility has different connotations as well and we need not confuse the reader by focusing on that. I'd recommend the following for communicating CODA's vision/mission/ambition :

__Make open source beginner-friendly__

Our objective is to provide a model and a space for open source and documentation.

Followed by the existing content:

We focus on documentation contributions because they present an opportunity for the public to share their direct experience with software to its maintainers, it has the lowest barrier to entry, it is low risk, and because documentation is always evolving and growing in projects. By contributing to documentation, members get closer to the software itself, which is good preparation if they want to engage as developers, too.
...

shaloo reviewed (commented):
A few comments.
